tox.ini

```diff
@@ -1,67 +1,68 @@
 [tox]
+# skip install to avoid rebuilding the world, do the install with setup.py
+# instead of pip below
+skip_install = True
 # these are the environments that will run when you
 # execute `tox` in the command-line
-# Note: linting is not included by default
 envlist =
     lint
+    build
     all-deps
     min-deps
     examples
-
-
-[testenv:all-deps]
-# skip install to avoid rebuilding the world, do the install with setup.py
-# instead of pip below
-skip_install = True
 passenv =
     RASCALINE_BUILD_TYPE
     RUSTUP_HOME
     CARGO*
 
+[testenv:all-deps]
 deps =
     discover
     numpy
     ase
     chemfiles
-
 commands =
     python setup.py --quiet install
     discover -p "*.py" -s python/tests
 
 [testenv:min-deps]
-skip_install = True
-passenv =
-    RASCALINE_BUILD_TYPE
-    RUSTUP_HOME
-    CARGO*
-
 deps =
     discover
     numpy
-
 commands =
     python setup.py --quiet install
     discover -p "*.py" -s python/tests
 
 [testenv:examples]
-skip_install = True
-passenv =
-    RASCALINE_BUILD_TYPE
-    RUSTUP_HOME
-    CARGO*
-
 deps =
     numpy
     chemfiles
     ase
-
 commands =
     python setup.py --quiet install
     python python/tests/check_examples.py
 
+[testenv:build]
+allowlist_externals =
+    cd
+    cmake
+    make
+    mkdir
+deps =
+    setuptools
+    wheel
+    twine
+    check-manifest
+skip_install = true
+commands =
+    mkdir -p build && cd build && cmake .. && make install
```

Comment:
In addition to what was in the issue, it would be a good idea to try to build a small example cmake project using the installed library, making sure the installed files are actually usable.

This might require some more filesystem management, with a `CMAKE_INSTALL_PREFIX=<somepath/somewhere>`, and pointing cmake there when building the example project.

---

Also, instead of using make which is not available everywhere I would use `cmake --build . --target install`. Depending on how familiar you are with cmake, I'm happy to help you do something like this or do it myself.

Reply:
I am not a expert in cmake building but I can specify a path.